storage/scylladb_identity.c

```diff
@@ -289,7 +282,28 @@ status_t db_insert_identity_table(db_client_service_t* service, db_identity_t* o
     ta_log_error("%s\n", "prepare INSERT query fail");
     return SC_STORAGE_CASSANDRA_QUREY_FAIL;
   }
-  statement = ret_insert_identity_statement(insert_prepared, obj);
+  statement = cass_prepared_bind(insert_prepared);
+  if (cass_statement_bind_bytes_by_name(statement, "hash", obj->hash, DB_NUM_TRYTES_HASH) != CASS_OK) {
+    ta_log_error("Fail to bind value to cass_statement\n");
+    ret = SC_STORAGE_CASSANDRA_QUREY_FAIL;
```

Comment:
Should we replace with the code like
```
ret = SC_STORAGE_CASSANDRA_QUREY_FAIL;
ta_log_error("%s\n", ta_error_to_string(ret));
```

It would make the log message less flexible for different situation but more unified?

Reply:
I prefer to show more details for troubleshooting.
Many places in TA code base using `ta_error_to_string` actually show incorrect/inaccurate messages.

Reply:
I think this should be discussed in the future.